### PR TITLE
bootctl: add --random-seed=yes/no

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: man/bootctl.xml [hunk 1]
```diff
@@ -385,6 +385,15 @@
         <xi:include href="version-info.xml" xpointer="v220"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--random-seed=yes|no</option></term>
+        <listitem><para>By default the <command>install</command>command initializes a random seed file in
+        the ESP. When creating an image it may be desirable to disable that in order to avoid having the
+        same seed in all instances</para>
```
Comment (yuwata) [inline man/bootctl.xml]: Missing period at the end.

File: man/bootctl.xml [hunk 2]
```diff
@@ -385,6 +385,15 @@
         <xi:include href="version-info.xml" xpointer="v220"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--random-seed=yes|no</option></term>
+        <listitem><para>By default the <command>install</command>command initializes a random seed file in
```
Comment (bluca) [inline man/bootctl.xml]: missing space before "command"

File: man/bootctl.xml [hunk 3]
```diff
@@ -385,6 +385,15 @@
...
+        the ESP. When creating an image it may be desirable to disable that in order to avoid having the
+        same seed in all instances</para>
+
+        <xi:include href="version-info.xml" xpointer="v227"/></listitem>
```
Comment (bluca) [inline man/bootctl.xml]: 257

**Review by bluca — COMMENTED** [review]
Update `shell-completion/bash/bootctl` too please

File: man/bootctl.xml (hunk 1 above)
Comment (lnussel) [reply man/bootctl.xml]: fixed

File: man/bootctl.xml (hunk 2 above)
Comment (lnussel) [reply man/bootctl.xml]: fixed

File: man/bootctl.xml (hunk 3 above)
Comment (lnussel) [reply man/bootctl.xml]: fixed

**Review by bluca — APPROVED** [review]

**Comment by poettering** [comment]
lgtm, too

File: src/boot/bootctl.c [hunk 4]
```diff
@@ -334,6 +339,14 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_touch_variables = false;
                         break;
 
+                case ARG_RANDOM_SEED:
+                        r = parse_boolean_argument("--random-seed=", optarg, NULL);
```
Comment (yuwata) [inline src/boot/bootctl.c]: Why not to pass `arg_install_random_seed` here?

**Review by yuwata — APPROVED** [review]

**Pull request merged**